app/scripts/filters/resources.js

```diff
@@ -1084,12 +1089,16 @@ angular.module('openshiftConsole')
       DeploymentConfig: 'dc',
       ImageStream: 'is',
       Pod: 'pod',
+      ReplicaSet: 'rs',
       ReplicationController: 'rc',
       Route: 'route',
       Service: 'svc',
     };
     return function(kind) {
-      return abbreviated[kind] || kind;
+      if (!kind) {
+        return kind;
+      }
+      return abbreviated[kind] || kind.toLowerCase();
```

Comment:
It's used for the events sidebar for a specific involved object like "rc/foo" or "deployment/bar"